src/app/components/stepOne/NewFollow.tsx

```diff
@@ -0,0 +1,15 @@
+import style from './NewFollow.module.css'
+import { useStepsContext, STEPS_NAMES } from '@/app/context/stepsContext'
+
+export function NewFollow() {
```

Comment:
```suggestion
export function NewFollowUpAppointment() {
```
best to avoid being cryptic with naming components - the name should make sense to future-you and other devs who might work on the code